src/layers/basic.jl

```diff
@@ -253,3 +253,47 @@ end
 function Base.show(io::IO, b::SkipConnection)
   print(io, "SkipConnection(", b.layers, ", ", b.connection, ")")
 end
+
+"""
+    Parallel(connection, layers...)
+
+Create a 'Parallel' layer that passes an input array to each path in
+`layers`, reducing the output with `connection`.
+
+Equivalent to calling `reduce(connection, [l(x) for l in layers]...)`.
+
+# Examples
+
+```jldoctest
+julia> model = Chain(Dense(1, 1), Parallel(vcat, Dense(1, 1), Dense(1, 3), Chain(Dense(1, 5), Dense(5, 2))), Dense(6, 1));
+
+julia> size(model(rand(1)))
+(1,)
```

Comment:
May I suggest making these numbers all distinct?
```suggestion
julia> model = Chain(Dense(3, 13, tanh), Parallel(vcat, Dense(13, 4), Chain(Dense(13, 7, tanh), Dense(7, 4))), Dense(8, 17));

julia> size(model(rand(3)))
(17,)
```